src/platform/intel/ace/platform.c

```diff
@@ -80,18 +81,22 @@ static struct pm_notifier pm_state_notifier = {
 	.state_exit = cpu_notify_state_exit,
 };
 
+#if CONFIG_KCPS_DYNAMIC_CLOCK_CONTROL
+/* Value to be determined experimentaly */
+#define BASE_CPS_USAGE 10000
+#else
+#define BASE_CPS_USAGE (CLK_MAX_CPU_HZ / 1000)
+#endif
+
 /* Runs on the primary core only */
 int platform_init(struct sof *sof)
 {
 	int ret;
 
 	trace_point(TRACE_BOOT_PLATFORM_CLOCK);
 	platform_clock_init(sof);
-
-	/* Set DSP clock to MAX using KCPS API. Value should be lowered when KCPS API
-	 * for modules is implemented
-	 */
-	ret = core_kcps_adjust(cpu_get_id(), CLK_MAX_CPU_HZ / 1000);
+	kcps_budget_init();
+	ret = core_kcps_adjust(cpu_get_id(), BASE_CPS_USAGE);
```

Comment:
The commit message does not match with the change..
If I'm not mistaken we already registered a 'base FW consumption' which happened to be resulting full speed, right?

The commit is more like to change the base FW consumption to a lower, yet to be determined number?

Reply:
Well, for me its fine because actually we are now registering proper **basefw** consumption. Previously it wasnt **basefw** consumption, it was just the highest possible value.
Anyway updated commit a little to note about proper basefw value